pkg/minikube/registry/registry_test.go

```diff
@@ -39,41 +53,62 @@ func TestRegistry(t *testing.T) {
 	}
 
 	registry := createRegistry()
+	t.Run("registry.Register", func(t *testing.T) {
+		t.Run("foo", func(t *testing.T) {
+			if err := registry.Register(foo); err != nil {
+				t.Fatal("expect nil")
+			}
+		})
+		t.Run("fooAlreadyExist", func(t *testing.T) {
+			if err := registry.Register(foo); err != ErrDriverNameExist {
+				t.Fatal("expect ErrDriverNameExist")
+			}
+		})
+		t.Run("bar", func(t *testing.T) {
+			if err := registry.Register(bar); err != nil {
+				t.Fatal("expect nil")
+			}
+		})
+	})
+	t.Run("registry.List", func(t *testing.T) {
+		list := registry.List()
+		if len(list) != 2 {
+			t.Fatalf("expect len(list) to be %d; got %d", 2, len(list))
+		}
+		if !(list[0].Name == "bar" && list[1].Name == "foo" || list[0].Name == "foo" && list[1].Name == "bar") {
```

Comment:
break the `||` will read better with two ifs

Reply:
splitted to 2 lines. Hope it's good enough.